pkg/collectors/aws/asg/asg.go

```diff
@@ -160,21 +159,17 @@ func (h *handler) Run(ctx context.Context) error {
 }
 
 func (h *handler) runOnce(ctx context.Context) error {
-	result, err := h.sqs.ReceiveMessageWithContext(ctx, &sqs.ReceiveMessageInput{
-		AttributeNames: []*string{
-			aws.String("All"),
-		},
-		MessageAttributeNames: []*string{
-			aws.String(sqs.QueueAttributeNameAll),
-		},
-		QueueUrl:            aws.String(h.url),
-		MaxNumberOfMessages: aws.Int64(10),
-		VisibilityTimeout:   aws.Int64(20),
-		WaitTimeSeconds:     aws.Int64(10),
+	result, err := h.sqs.ReceiveMessage(ctx, &sqs.ReceiveMessageInput{
+		AttributeNames:        []types.QueueAttributeName{types.QueueAttributeNameAll},
+		MessageAttributeNames: []string{"All"},
```

Comment:
Weird that they switched the parameters for the attributes here, but it's correct:
https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/sqs#ReceiveMessageInput